utils/utils.jsx

```diff
@@ -2019,3 +2021,12 @@ export function stringToNumber(s) {
 
     return parseInt(s, 10);
 }
+
+export function shouldProcessApps(state) {
+    const statuses = state.entities.admin.pluginStatuses;
+    let proxy;
+    if (statuses) {
+        proxy = statuses['com.mattermost.apps']; // eslint-disable-line no-unused-vars
+    }
+    return true;//Boolean(proxy && proxy.state === PluginStates.PLUGIN_STATE_RUNNING);
```

Comment:
Can we have this fixed before this PR is merged?

Reply:
https://community.mattermost.com/core/pl/hqobzi34dpb5mkmtefc8wa8z6h

Reply:
I added the feature flag code commented, since the feature flag is still not created. If you are OK with it, we can merge.